packages/messenger-widget/src/components/Contacts/bl.ts

```diff
@@ -86,7 +89,7 @@ export const onContactSelected = (
     // add click css
     event.currentTarget.classList.add(classTwo);
     const actionItem: any = document.getElementById('contact-' + index);
-    actionItem.style.display = 'none';
+    // actionItem.style.display = 'none';
```

Comment:
Do you intend to keep using that var ? Otherwiese please remove the comment

Reply:
Oh yes, I will remove that